dotcom-rendering/src/web/layouts/StandardLayout.tsx

```diff
@@ -458,7 +458,7 @@ export const StandardLayout = ({ CAPIArticle, NAV, format }: Props) => {
 				<AdSlot position="survey" display={format.display} />
 			)}
 
-			<main>
+			<main data-layout="StandardLayout">
```

Comment:
It's annoying that it seems so hard to get the name of a component from inside itself

Reply:
Yeah this is frustrating, but I think for one use per layout there's not even much value in trying to find an abstraction for it

Reply:
Agreed

Reply:
Agree on the abstraction - it feels like it would be fiddly. I wonder if we could write a test to ensure all layouts have a name for people adding a future one to be prompted and to describe why it's there, but I'm not sure if that's any less fiddly.